File: configlibrary/src/main/java/com/shmedo/configlibrary/iot/model/adme/AdmeStepperMotorInfo.java
package com.shmedo.configlibrary.iot.model.adme;

import android.text.TextUtils;

/**
 * 创建者:   gonghe <br/>
 * 创建时间:  12/28/20 <br/>
 * 描述：    ADME 步进电机参数
 */
public class AdmeStepperMotorInfo {
    private String posnegtest;//正反测（0:关闭，1:开启）
    private String absprsion;//绝对精度修正值
    private String movspeed;//电机运动速度

    public String getPosnegtest() {
        return TextUtils.isEmpty(posnegtest) ? "" : posnegtest;
    }

    public void setPosnegtest(String posnegtest) {
        this.posnegtest = posnegtest;
    }

    public String getAbsprsion() {
        return TextUtils.isEmpty(absprsion) ? "" : absprsion;
    }

    public void setAbsprsion(String absprsion) {
        this.absprsion = absprsion;
    }

    public String getMovspeed() {
        return TextUtils.isEmpty(movspeed) ? "" : movspeed;
    }

    public void setMovspeed(String movspeed) {
        this.movspeed = movspeed;
    }
}

File: lib_device_base/src/main/java/com/shmedo/lib/device/base/iot_cmd/model/adme/AdmeStepperMotorInfo.kt
package com.shmedo.lib.device.base.iot_cmd.model.adme

/**
 * 创建者:   gonghe <br></br>
 * 创建时间:  12/28/20 <br></br>
 * 描述：    ADME 步进电机参数
 */
data class AdmeStepperMotorInfo(
    var posnegtest: String = "", //正反测（0:关闭，1:开启）
    var absprsion: String = "", //绝对精度修正值
    var movspeed: String = "", //步进电机运动速度
    var movesm: String = "", //步进电机力矩
)